lib/pure/asyncftpclient.nim

```diff
@@ -79,7 +79,13 @@ proc connect*(ftp: AsyncFtpClient) {.async.} =
     # 120 Service ready in nnn minutes.
     # We wait until we receive 220.
     reply = await ftp.expectReply()
-  assertReply(reply, "220")
+
+  # Handle 220 messages from the server
+  if reply.startsWith("220"):
+    assertReply(reply, "220")
+    while reply.continuesWith("-", 3): # handle multiline 220 message
```

Comment:
What is the actual message that the server sends? Why are you checking for a dash at position 3?